woocommerce/woocommerce-paypal-payments
# Update the "standard payments" tab settings

**Issue**: #1065 

---

### Description

Just like the "Pay Later" tab, the "Standard Payments" tab settings should be also improved. 

The PR will:

- Create selectbox for selecting locations.
- Remove "enable" smart buttons for each location settings.
- Add the "styling per location" checkbox.
- Refactor JS to work according to new settings.

### Steps to Test




1. Go to the "Standard Payments" tab settings.
2. Observe the "Smart Buttons" settings.

---

Closes #1065 .
